submissions/FoxYulya/HTML_CSS_Practice/index.html

```diff
@@ -0,0 +1,166 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooli-style Popup</title>
+    <link rel="shortcut icon" href="img/popup-icons/favicon.png" type="image/x-icon">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>                   
+    <div class="wrapper">
+        <header class="header">
+            <nav class="main-menu">
+                <ul>
+                    <li><a href="#">Mail</a></li>
+                    <li><a href="#">Images</a></li>
+                 </ul>
+            </nav>  
+            <input class='popup_checkbox' type="checkbox" name="" id="popup_checkbox" tabindex="1">
```

Comment:
All menu should be element of navigation list, and when you navigate with keyboard focus should appear on `menu icon` just after `images` link.